src/main/java/com/google/cloud/tools/libraries/libraries.json

```diff
@@ -0,0 +1,31 @@
+[
+  {
+    "name": "Google BigQuery API",
+    "documentation": "https://cloud.google.com/bigquery/docs/reference/rest/v2/",
```

Comment:
Totally get it. If you want to find that information you can click into any of the other links we provide and get there from the big query documentation left nav.  Three levels of links per API in the wizard is more than enough.